automation/Dockerfile

```diff
@@ -13,30 +13,22 @@ RUN (cd /lib/systemd/system/sysinit.target.wants/; for i in *; do [ $i == \
     rm -f /lib/systemd/system/basic.target.wants/*;\
     rm -f /lib/systemd/system/anaconda.target.wants/*;
 
-RUN yum -y install iproute NetworkManager epel-release git cairo* && \
-    yum -y install python2-pip gcc python-devel gobject-introspection-devel && \
+# Patch with NetworkManager fix for BZ#1642625
+RUN yum-config-manager --add-repo https://copr.fedorainfracloud.org/coprs/till/nmstate-NetworkManager/repo/epel-7/till-nmstate-NetworkManager-epel-7.repo && \
+    yum-config-manager --enable till-nmstate-NetworkManager && \
+    yum -y install NetworkManager
+
+RUN yum-config-manager --add-repo http://people.redhat.com/fge/el7_pytest/fge_el7_pytest.repo && \
```

Comment:
I would prefer to use COPR instead of the repository in the people page. That way the packages will also be GPG signed.

About the organization name, to use nmstate on COPR I suppose we need a Fedora user with this name that has access to COPR. Not sure, if this is really worth the trouble to maintain an extra user account since these are just internal details.

Reply:
Yes, it's public accessible.
I will try to find a better place for it.